doc/source/getting-started/qibosoq_lab_cal.rst

```diff
@@ -0,0 +1,486 @@
+Qibosoq - Qibolab - Qibocal
+===========================
+
+In these examples, we will see how to perform some basic qubit calibration experiments using the three levels of abstraction provided by the Qibo ecosystem:
+
+- Qibosoq: basic pulse level and direct RFSoC connection
+- Qibolab: higher pulse level and integrated control of the complete setup
+- Qibocal: experiment level
```

Comment:
From just this text is not fully clear that the sections below are doing the same thing in three different ways, since to an unexperienced reader it might seem like you need to operate in all the three of them at the same time.

I would just spend a few more words on the layout ("For each experiment we will show a parallel implementation of the same operation, resulting in the same (or very similar) plot", or any better wording you can think about, even integrated in the first sentence), and I would briefly describe what is being done in the code at the beginning of each experiment.